adapters/auto_for_causal_lm/dataloop.json

```diff
@@ -3,10 +3,12 @@
   "description": "AutoForCausalLM for chatting - HF",
   "attributes": {
     "Provider": "Hugging Face",
```

Comment:
HF provider?

Reply:
In this case, yes, because it's a generic adapter using HF classes